openedx/core/djangoapps/credit/migrations/0007_auto__del_field_crediteligibility_provider.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this migration may conflict with what's currently in master.  Please make sure you rebase, delete this migration, then regenerate migrations.